manifests/hpo-operator-deployment.yaml_template

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hpo
+  labels:
+    app: hpo
+spec:
+  replicas: 1
+  selector:
+      matchLabels:
+        name: hpo
+  template:
+    metadata:
+      labels:
+        app: hpo
+        name: hpo
+        operatorframework.io/os.linux: supported
+    spec:
+      serviceAccountName: hpo-sa
+      containers:
+      - name: hpo
+        image: "{{ HPO_IMAGE }}"
+        imagePullPolicy: Always
+        ports:
+         - name: hpo-port
+           containerPort: 8085
```

Comment:
Why is the logging level missing from here? This is needed so users can easily change logging level while deploying. This also means that we need the corresponding config file where a user can specify the logging level
```
        env:
          - name: LOGGING_LEVEL
            valueFrom:
              configMapKeyRef:
                name: autotune-config
                key: logging_level
                optional: true
```

Reply:
Added corresponding files for docker and minikube